gacl/gacl.ini.php

```diff
@@ -1,6 +1,6 @@
 ;<?php trigger_error('This is a hidden INI file, not PHP.  Go away!', E_USER_ERROR); ?>
 ;
-debug 			= FALSE
+debug 			= TRUE
```

Comment:
haven't been able to view any gacl debug output in the php error log

Reply:
this should throw gacl output in the setup.php script